packages/npm/@amazeelabs/gatsby-silverback-cloudinary/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@amazeelabs/gatsby-silverback-cloudinary",
+  "version": "1.0.0",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@amazeelabs/eslint-config": "^1.4.36",
+    "@amazeelabs/jest-preset": "^1.3.40",
+    "@amazeelabs/prettier-config": "^1.1.1",
+    "@types/crypto-js": "^4.1.1",
+    "@types/jest": "29.2.1",
+    "@types/node": "16.18.11",
+    "eslint": "8.26.0",
+    "gatsby": "~3.3.0",
+    "jest": "29.2.2",
```

Comment:
We use vitest nowadays (although `@amazeelabs/scaffold` still has to be updated). Replacing it would simply mean removing jest and its config, installing vitest and adding `import { describe, it, expect } from 'vitest';` to the test file.

Reply:
we should use now vitest